### PR TITLE
Fix: broken assets path gh-pages

### PR DESCRIPTION
**Pull request by hblitza**
This adapts the base path in index.ejs template before building the gh-pages.  
Other Alternatives would be, e.g.
  - extra webpack-common for gh-pages

What do you think @terrestris/devs ?

### CODE REVIEW

File: .github/workflows/on-release.yml [hunk 1]
```diff
@@ -28,6 +28,9 @@ jobs:
     - name: Install dependencies ⏬
       run: npm ci
 
+    - name: Adapt base path for gh-pages :twisted_rightwards_arrows:
```
Comment (dnlkoch) [inline .github/workflows/on-release.yml:31]:
```suggestion
    - name: Adapt base path for gh-pages 🔀
```

File: .github/workflows/on-release.yml [hunk 2]
```diff
@@ -28,6 +28,9 @@ jobs:
...
+      run: sed -i 's/\htmlWebpackPlugin.options.appPrefix/\/latest\//' resources/public/index.ejs
```
Comment (dnlkoch) [inline .github/workflows/on-release.yml:32]: Basically +1, have you tested the updated path on the `gh-pages` branch beforehand?

Comment (dnlkoch) [inline .github/workflows/on-release.yml:32]: Basically +1, have you tested the updated path on the `gh-pages` branch beforehand?

**Comment by dnlkoch** [comment]
@hblitza [Currently](https://terrestris.github.io/shogun-demo-client/latest/) it looks like the assets can be loaded without any issues. Can you confirm that? If so, is this fix still needed?

**Comment by hblitza** [comment]
Yes, I can confirm this. Thanks, will close this.

**Pull request closed without merging**